packages/application-shell/src/configure-apollo.js

```diff
@@ -64,23 +64,24 @@ const fragmentMatcher = new IntrospectionFragmentMatcher({
   introspectionQueryResultData,
 });
 
+const typeNamesWithoutIdAsIdentifier = ['Project', 'BaseMenu', 'NavbarMenu'];
```

Comment:
I hope naming this in a variable makes more clear why we do this while keeping the code below more concise.